app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -19,28 +19,32 @@
           class="select-role">
           <ui-select-match placeholder="Select a storage class">
             <span>
-              {{$select.selected.metadata.name}}
+              {{$select.selected.metadata.name}}<br/>
             </span>
           </ui-select-match>
           <ui-select-choices
             repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } ">
             <div>
               <span ng-bind-html="sclass.metadata.name  | highlight : $select.search"></span>
-              <span ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description)" class="text-muted">
-                <small>&ndash;
+              <div ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description) || (sclass | annotation : 'storage.alpha.openshift.io/access-mode')" class="text-muted">
+                <small>&nbsp;
                   <span ng-if="sclass.parameters.type">
                     Type: {{sclass.parameters.type}}
                   </span>
                   <span ng-if="sclass.parameters.zone">
                     <span ng-if="sclass.parameters.type">|</span>
                     Zone: {{sclass.parameters.zone}}
                   </span>
-                  <span ng-if="sclass | description">
+                  <span ng-if="(sclass | annotation : 'storage.alpha.openshift.io/access-mode')">
```

Comment:
You have this check repeated so many times in the view that it's probably better as a filter.

```js
filter('storageClassAccessMode', function(annotationFilter) {
  return function(storageClass) {
    return annotationFilter(storageClass, 'storage.alpha.openshift.io/access-mode');
  };
});
```

Then everywhere in the view you can use

```html
<span ng-if="sclass | storageClassAccessMode">
```

This also keeps the annotation name in one place if we ever change it.